app/views/uploads/_data_criteria.html.erb

```diff
@@ -0,0 +1,130 @@
+    <% if dataCriteria[:definition] == 'satisfies_all' ||  dataCriteria[:definition] == 'satisfies_any' %>
+      <%= render partial: 'satisfies', locals: {reference: reference, measure:measure, tag: "span"} %>
+    <% else %>
+      <% if dataCriteria[:children_criteria]%>
+        <% if dataCriteria[:derivation_operator] == 'INTERSECT' || dataCriteria[:derivation_operator] == 'UNION'%>
+          <% unless expandVariable %>
+            <ul><li><span class="set-op {{dataCriteria.key}} rationale-target">
+              <%=dataCriteria[:derivation_operator]%> of:
+            </span></li>
+          <% end %>
+        <% end %>
+        <ul>
+          <% dataCriteria[:children_criteria].each do |child| %>
+            <% if (dataCriteria[:derivation_operator] == 'INTERSECT' ||dataCriteria[:derivation_operator] == 'UNION')%>
+              <% if !expandVariable %>
+                <li class="set-op-item">
+              <% else %>
+                <li>
+              <% end %>
+            <% else %>
+              <li>
+                <span class="conjunction {{../../dataCriteria.key}} rationale-target"><%= (dataCriteria[:derivation_operator] == 'XPRODUCT')? 'AND':'' %>: <span class="sr-only sr-highlight-status"></span></span>
+            <% end %>
+              <%= render partial: 'data_criteria', locals: {reference: child, expandVariable: false, measure:measure, tag: "span"} %>
+            </li>
+          <% end %>
+          <% if dataCriteria[:temporal_references] %>
+            <% dataCriteria[:temporal_references].each do |temp_ref| %>
+              <li>
+                <span class="{{../dataCriteria.key}} rationale-target">
+                  <%= render partial: 'temporal_reference', locals: {temporalReference: temp_ref, measure:measure, tag: "span"} %>
+                </span>
+              </li>
+            <% end %>
+          <% end %>
+        </ul>
+        <% if dataCriteria[:derivation_operator] == 'INTERSECT' || dataCriteria[:derivation_operator] == 'UNION'%>
+          <% unless expandVariable %>
+            </ul>
+          <% end %>
+        <% end %>
+
+      <% end %>
+    <% end %>
+  <% else %>
+    <span class="criteria-title">
+      <!-- {{#unless hideTitle}} -->
+      <% if dataCriteria[:specific_occurrence] && !(dataCriteria[:description].include? "Occurrence")%>
+        Occurrence <%= dataCriteria[:specific_occurrence] %>:
+      <% end %>
+      <% if dataCriteria[:variable] && !(dataCriteria[:description].include? "$")%>$<% end %><%= dataCriteria[:description] %>
+      <!-- {{/unless}} -->
+
+      <% if dataCriteria[:value] && dataCriteria[:type]!= 'characteristic'%>
+        <%= render partial: 'value', locals: {value:dataCriteria[:value], rangeComparison: "", measure:measure, tag: "span"} %>
+      <% end %>
+
+      <% if dataCriteria[:field_values] %>
+        <% dataCriteria[:field_values].each do |key,value| %>
+          <!--ANYNonNull type means field value type -->
```

Comment:
Since this will be in the generated HTML, it should be removed, or if you want to keep it as a developer comment, you can convert it to an ERB comment rather than an HTML one, i.e.
`<%# comment here %>`